### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by saikrishnametpalli-nttdata**
# Description

This PR includes the following proposed change(s):

- 2.0.2

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [x] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/src/assets/app.config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "production": false,
   "environment": "dev",
-  "version": "2.0.0",
+  "version": "2.0.2",
```
Comment (KDNeufeld) [inline src/frontend/citizen-portal/src/assets/app.config.json:4]: I think this is now 0.0.0 in main to be clear that version numbers come from helm, not the code base.

File: src/frontend/staff-portal/src/assets/app.config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "production": false,
   "environment": "dev",
-  "version": "2.0.0",
+  "version": "2.0.2",
```
Comment (KDNeufeld) [inline src/frontend/staff-portal/src/assets/app.config.json:4]: Same. 0.0.0

File: src/frontend/citizen-portal/src/assets/app.config.json (hunk 1 above)
Comment (saikrishnametpalli-nttdata) [reply src/frontend/citizen-portal/src/assets/app.config.json:4]: I will change this

**Review by KDNeufeld — APPROVED** [review]

**Review by pbolduc — APPROVED** [review]
LGTM

**Pull request merged**